MTM-49948: added note, that it is possible to confirm POP by support (#716)
SoftwareAG/c8y-docs: content/device-integration/mqtt-bundle/device-certificates.md

- The proof of possession is confirmed if the uploaded signed verification code matches the signed verification code expected by the platform:
![Proof of possession completed](/images/mqtt/devmgmt-mqtt-cert-pop-completed.png)

{{< c8y-admon-info >}}

> If it is not possible for the administrator to carry out this process on their own for organizational reasons, they can manually request the proof of possession for the corresponding certificate and the {{< product-c8y-iot >}} support team can complete the proof of possession through a back end API upon reasonable verification.

Reviewer: ```suggestion
If administrators cannot carry out this process on their own for organizational reasons, they can manually request the proof of possession for the corresponding certificate and the {{< product-c8y-iot >}} support team can complete the proof of possession through a back end API upon reasonable verification.
```

- The proof of possession is confirmed if the uploaded signed verification code matches the signed verification code expected by the platform:
![Proof of possession completed](/images/mqtt/devmgmt-mqtt-cert-pop-completed.png)

> Note: If it is not possible for the admin to carry out this process himself for organizational reasons, the proof of possession for the corresponding certificate can be manually requested and the {{< product-c8y-iot >}} support team can complete the proof of possession through a backend API upon reasonable verification.

Reviewer 2: ```suggestion
{{< c8y-admon-info >}}
If it is not possible for the administrator to carry out this process on their own for organizational reasons, they can manually request the proof of possession for the corresponding certificate and the {{< product-c8y-iot >}} support team can complete the proof of possession through a back end API upon reasonable verification.
{{< /c8y-admon-info >}}
```
